intranet/middleware/access_log.py

```diff
@@ -1,11 +1,13 @@
 import logging
+
```

Comment:
Both of these are standard library imports; they don't need a blank line separating them.
```suggestion
```